CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG
 
+- Remove `areas` field from BusinessSupportEdition
```

Comment:
We generally only add something to the `CHANGELOG` when we're doing the version bump. If you remove this I'll merge it and then you can bump the version and add this note.